cypress.config.js

```diff
@@ -22,7 +22,7 @@ module.exports = defineConfig({
             description: faker.lorem.words(),
             body: faker.lorem.words(),
             tag: faker.lorem.word()
-          };;
+          };
```

Comment:
The `generateArticle` task correctly uses the `faker` library to generate random article data, which is in line with the task requirement to use faker for generating fake data.